src/stores/compliance-pci.md

```diff
@@ -3,89 +3,24 @@ title: PCI Compliance Guidelines
-            </td>
-            <td>Do not use vendor-supplied defaults for system passwords and other security parameters.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Protect stored cardholder data.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Encrypt transmission of cardholder data across open, public networks.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Use and regularly update antivirus software.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Develop and maintain secure systems and applications.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Restrict access to cardholder data by business need to know.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Assign a unique ID to each person with computer access.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Restrict physical access to cardholder data.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Track and monitor all access to network resources and cardholder data.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Regularly test security systems and processes.</td>
-         </tr>
-         <tr>
-            <td>
-               <img src="{{ site.baseurl }}{% link images/images/assets/checkbox.png %}" class="checkbox" />
-            </td>
-            <td>Maintain a policy that addresses information security.</td>
-         </tr>
-      </tbody>
-   </table>
-To learn more, see: [Magento Approach to PCI Compliance][1].
+|![checkbox]({{ site.baseurl }}{% link images/images/assets/checkbox.png %})|Install and maintain a firewall configuration to protect cardholder data.|
```

Comment:
This table isn't rendering in the Github preview.

Reply:
This one has checkbox images in the first column, and GitHub does not like that. But is does render correctly in the staging build.